CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Fixed
-
+- Fix typo on the build instructions of the README
```

Comment:
Are you up-to-date with the upstream `main`? There have been changes to the CHANGELOG since. Namely, I released v2.1.0 this morning, but I cannot see this version in your Makefile. I think you will need to rebase with us before updating the CHANGELOG.

Reply:
Got it, updating my local to the latest.